### PR TITLE
Fixes #17144: Improve doc related to plugins/subscriptions

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/17144

### CODE REVIEW

**Comment by amousset** [comment]
do not merge

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
ready for review

**Comment by amousset** [comment]
PR rebased

**Comment by amousset** [comment]
PR updated with a new commit

File: src/reference/modules/installation/pages/dev.adoc [hunk 1]
```diff
@@ -26,3 +26,6 @@ provide feedback:
 
 To get the full list of repository URLs, go
 to https://docs.rudder.io/versions[the page presenting Rudder versions].
+
+You can then follow stadard install documentation, you only need to replace
```
Comment (Fdall) [inline src/reference/modules/installation/pages/dev.adoc:30]: standard

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**